include/coreinit/memorymap.h

```diff
@@ -26,6 +26,12 @@ typedef enum OSMemoryMapMode
 uint32_t
 OSEffectiveToPhysical(uint32_t virtualAddress);
 
+uint32_t
+__OSPhysicalToEffectiveCached(uint32_t effectiveAddress);
```

Comment:
The  `effectiveAddress` parameter should probably be `pyhsicalAddress` here. The function translates a physical address to a virtual/effective address.